### PR TITLE
Integrate irmin-chunk and its tests

### PR DESCRIPTION
**Pull request by g2p**

### CODE REVIEW

File: src/irmin-chunk/_oasis [hunk 1]
```diff
@@ -0,0 +1,35 @@
+OASISFormat: 0.4
```
Comment (samoht) [inline src/irmin-chunk/_oasis]: can you remove the oasis file? This is not needed as you converted the build to use jbuilder

File: irmin-chunk.opam [hunk 2]
```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Mounir Nasr Allah" "Thomas Gazagnaire"]
+license:      "ISC"
+homepage:     "https://github.com/mirage/irmin-chunk"
+bug-reports:  "https://github.com/mirage/irmin-chunk/issues"
+dev-repo:     "https://github.com/mirage/irmin-chunk.git"
+
+build: [
+  ["./configure" "--prefix" prefix]
```
Comment (samoht) [inline irmin-chunk.opam]: that build command is incorrect, could you copy the one in other opam files?

Comment (g2p) [inline irmin-chunk.opam]: Whoops, done.

**Comment by samoht** [comment]
Thanks!

**Pull request merged**